README.md

```diff
@@ -3,6 +3,30 @@
 
 kafka-connect-venafi-tpp is a [Kafka connector](http://kafka.apache.org/documentation.html#connect) for Venafi Trust Protection Platform security events.
 
+
+Description
+---
+This connector connects via HTTP to your instance of the Venafi Trust Protection Platform ( which shall be refered to TPP from here on) and pulls your Log events into kafka allowing you to do any filtering/transforming/processing you'd like to do within a comfortable kafka environment.  
+ **N.B.** Currently the connector starts from the of the beginning of time, a future release will allow the option of starting from now.
+
+EventLog object produced by this source **Connector** as defined by the Venafi WebSDK
```

Comment:
```suggestion
**EventLog**: object produced by this source Connector as defined by the Venafi WebSDK.
```

Reply:
The line is meant to be more of a header.